tron/core/recovery.py

```diff
@@ -26,9 +31,13 @@ def launch_recovery_actionruns_for_job_runs(job_runs, master_action_runner):
             log.info(f"Skipping recovery of {run} with no action runs (may have been cleaned up)")
             continue
 
-        ssh_runs, mesos_runs = filter_action_runs_needing_recovery(run._action_runs)
+        # Why do we do this separately if we just need to call recover()
```

Comment:
i've actually wondered this myself - we should probably look at this code once we're cleaning up all the mesos stuff and see if there's a real reason :)